daprdocs/content/en/reference/components-reference/supported-state-stores/_index.md

```diff
@@ -26,38 +26,41 @@ The following stores are supported, at various levels, by the Dapr state managem
-| [RethinkDB]({{< ref setup-rethinkdb.md >}})                                                      | ✅   | ✅                  | ✅    | ❌ | ✅ | Alpha  | v1 | 1.0 |
-| [Zookeeper]({{< ref setup-zookeeper.md >}})                    | ✅   | ❌                  | ✅    | ❌ | ❌ | Alpha  | v1 | 1.0 |
+| Name                                               |CRUD|Transactional|ETag| [TTL]({{< ref state-store-ttl.md >}}) | [Actors]({{< ref howto-actors.md >}}) | [State Query API]({{< ref howto-state-query-api.md >}}) | Status | Component version | Since |
+|----------------------------------------------------|----|-------------|----|----|----|----|-------|----|-----|
+| [Aerospike]({{< ref setup-aerospike.md >}})        | ✅ | ❌          | ✅  | ❌ | ❌ | ❌ | Alpha  | v1 | 1.0 |
+| [Apache Cassandra]({{< ref setup-cassandra.md >}}) | ✅ | ❌          | ❌  | ✅ | ❌ | ❌ | Alpha  | v1 | 1.0 |
+| [Cloudstate]({{< ref setup-cloudstate.md >}})      | ✅ | ❌          | ✅  | ❌ | ❌ | ❌ | Alpha  | v1 | 1.0 |
+| [Couchbase]({{< ref setup-couchbase.md >}})        | ✅ | ❌          | ✅  | ❌ | ❌ | ❌ | Alpha  | v1 | 1.0 |
+| [Hashicorp Consul]({{< ref setup-consul.md >}})    | ✅ | ❌          | ❌  | ❌ | ❌ | ❌ | Alpha  | v1 | 1.0 |
+| [Hazelcast]({{< ref setup-hazelcast.md >}})        | ✅ | ❌          | ❌  | ❌ | ❌ | ❌ | Alpha  | v1 | 1.0 |
+| [Memcached]({{< ref setup-memcached.md >}})        | ✅ | ❌          | ❌  | ✅ | ❌ | ❌ | Alpha  | v1 | 1.0 |
+| [MongoDB]({{< ref setup-mongodb.md >}})            | ✅ | ✅          | ✅  | ❌ | ✅ | 🔶 | Stable | v1 | 1.0 |
+| [MySQL]({{< ref setup-mysql.md >}})                | ✅ | ✅          | ✅  | ❌ | ✅ | ❌ | Alpha  | v1 | 1.0 |
+| [PostgreSQL]({{< ref setup-postgresql.md >}})      | ✅ | ✅          | ✅  | ❌ | ✅ | ❌ | Alpha  | v1 | 1.0 |
+| [Redis]({{< ref setup-redis.md >}})                | ✅ | ✅          | ✅  | ✅ | ✅ | ❌ | Stable | v1 | 1.0 |
+| [RethinkDB]({{< ref setup-rethinkdb.md >}})        | ✅ | ✅          | ✅  | ❌ | ✅ | ❌ | Alpha  | v1 | 1.0 |
+| [Zookeeper]({{< ref setup-zookeeper.md >}})        | ✅ | ❌          | ✅  | ❌ | ❌ | ❌ | Alpha  | v1 | 1.0 |
 
 
 ### Amazon Web Services (AWS)
-| Name                                                           | CRUD | Transactional | ETag | [TTL]({{< ref state-store-ttl.md >}}) | [Actors]({{< ref howto-actors.md >}}) | Status | Component version | Since |
-|------------------------------------------------------------------|------|---------------------|------|-----|--------|-----|-----|-------|
-| [AWS DynamoDB]({{< ref setup-dynamodb.md>}})                                                      | ✅   | ❌                   | ❌   | ❌ | ❌   |  Alpha | v1 | 1.0 |
+| Name                                               | CRUD | Transactional | ETag | [TTL]({{< ref state-store-ttl.md >}}) | [Actors]({{< ref howto-actors.md >}}) | [State Query API]({{< ref howto-state-query-api.md >}}) | Status | Component version | Since |
```

Comment:
```suggestion
| Name                                               | CRUD | Transactional | ETag | [TTL]({{< ref state-store-ttl.md >}}) | [Actors]({{< ref howto-actors.md >}}) | [Query]({{< ref howto-state-query-api.md >}}) | Status | Component version | Since |
```